packages/backend/src/services/OrganizationService/OrganizationService.ts

```diff
@@ -169,4 +176,30 @@ export class OrganizationService {
             data,
         );
     }
+
+    async getTheme(user: SessionUser): Promise<Theme> {
+        return this.themeModel.getThemeByOrganizationId(user.organizationUuid);
+    }
+
+    async createTheme(
+        user: SessionUser,
+        data: { colours: string[] },
+    ): Promise<Theme> {
+        if (user.ability.cannot('create', 'Project')) {
+            // TODO Change permission
+            throw new ForbiddenError();
```

Comment:
This could be an admin only setting. Admins already are the only ones that can update organizations: https://github.com/lightdash/lightdash/blob/f622b706e44c3499c3d25e63731823d90fdcb7a7/packages/common/src/authorization/organizationMemberAbility.ts#L50-L57

so you could reuse this by checking whether `user.ability.cannot('update', 'Organization')`